custom/icds_reports/management/commands/rebuild_ccs_record_ucr.py

```diff
@@ -20,6 +20,27 @@ def get_ucr_config_and_document_store(self, indicator_config_id, case_type_of_xm
         )
         return config, document_store
 
+    def pull_superviosr_partition_data(self, supervisor_ids, month):
+        CHUNK = 400
+        i = 0
+        data = []
+        while i+CHUNK < len(supervisor_ids):
```

Comment:
FYI, there is a helper function in `dimagi.utils.chunked` for doing something like this that might be more appropriate to use

Reply:
fixed. That was easy. thanks for pointing to that.